src/heartbeats.cpp

```diff
@@ -46,14 +45,14 @@ void HeartBeatBackupInfo::setRunningCommand(Command *runningCommand)
     mRunningCommand = runningCommand;
 }
 
-HeartBeatBackupInfo::Status HeartBeatBackupInfo::status() const
+int HeartBeatBackupInfo::status() const
 {
     return mStatus;
 }
 
 double HeartBeatBackupInfo::progress() const
 {
-    return std::max(0., std::min(1., static_cast<double>(mTransferredBytes) / static_cast<double>(mTotalBytes)));
+    return 0.;
```

Comment:
Maybe the class can be abstract <-- methods like this could be pure virtual

Reply:
good point. Since everything else makes HeartBeatInfo usable, I think it'll be better to have some mProgress and a setter. Just in case we don't want to have an inherited version for CU's or whatever backup may exist in the future

Reply:
done